### ISSUE TITLE
Mandatory Attributes on Pool Creation

### ISSUE THREAD

**Issue by miesi** [1and1/dim#146]
In the Traffic Sentinel traffic accounting project we are designing some attributes which every dim pool needs to have so our automated accounting process can determine what pool traffic belongs to which market (product) segment.

Question 1: Would it be possible to make these attributes mandatory when creating a pool so a pool creator must provide the information? If it is possible to have the attributes set as mandatory, would it be possible to make this easier for the pool creator by providing this as a command-line addition? I'm thinking something like the following as it will be easier than having to remember to use option flags to set the attributes:

ndcli create pool P <options> [user presses enter]

Please provide the following four pool attributes:
a-company: <text> [user presses enter]
a-country: <text> [user presses enter]
a-origin: <text> [user presses enter]
a-product: <text> [user presses enter]
[normal process proceeds]

In other words, each attribute is a single line and the next line does not show up until the user presses enter. It would extend the pool creation process a bit, but would be much easier than having to remember to set these attributes through option tags.

Question 2: If it is not possible to make them mandatory, could the pool creation process add the attributes with a "dummy" value (like UNSET) so we could programmatically detect the new pools with invalid attribute settings?

The attributes every pool should have set:

a-company
a-country
a-origin
a-product
